test/fastify-env.test.js

```diff
@@ -259,3 +258,25 @@ tests.forEach(function (testConf) {
     makeTest(t, options, testConf.isOk, testConf.confExpected, testConf.errorMessage)
   })
 })
+
+t.test('should use custom config key name', async t => {
+  const schema = {
+    type: 'object',
+    required: [ 'PORT' ],
+    properties: {
+      PORT: {
+        type: 'integer',
+        default: 6666
+      }
+    }
+  }
+
+  const fastify = Fastify()
+  await fastify.register(fastifyEnv, {
```

Comment:
Can you avoid using async/await? This module still supports Node 6.

Reply:
Fixed